plugins/notifications-backend/src/service/router.ts

```diff
@@ -213,9 +253,9 @@ export async function createRouter(
       }
       opts.createdAfter = new Date(sinceEpoch);
     }
-    if (req.query.minimal_severity) {
+    if (req.query.minimum_severity) {
```

Comment:
These changes are conflicting with https://github.com/backstage/backstage/pull/23537 which is waiting for review